Prac07/guitarTest.py

```diff
@@ -0,0 +1,31 @@
+from Prac07.guitar import Guitar
+
+
+def main():
+
+    i = 0
+    guitars = []
+    while True:
+        name = input("Name: ")
+        if not name:
+            break
```

Comment:
I'm not certain how efficient using a break is but generally speaking, it's better to make it so that what causes the loop to stop is the stopper, for example;

Instead of:

"""    while True:
           name = input("Name: ")
           if not name:
               break
           ..."""

Have:

"""name = "a"
   while name != "":
       name = input("Name: ")
       ..."""

Or:

"""name = input("Name: ")
   while name != "":
       ...
       name = input("Name: ")"""